config/filesystems.php

```diff
@@ -57,10 +57,10 @@
 
         's3' => [
             'driver' => 's3',
-            'key' => env('S3_KEY'),
-            'secret' => env('S3_SECRET'),
-            'region' => 'us-east-1',
-            'bucket' => env('S3_BUCKET', 'ds-rogue-qa'),
+            'key' => env('AWS_ACCESS_KEY'),
+            'secret' => env('AWS_SECRET_KEY'),
```

Comment:
These are set [here](https://github.com/DoSomething/infrastructure/blob/0e39adb7f1c3a90c729a80724a07dcdbba958727/components/iam_app_user/main.tf#L27-L28) in our Terraform config.